src/shogun/kernel/Kernel.h

```diff
@@ -1064,9 +1064,9 @@ class CKernel : public CSGObject
 		bool lhs_equals_rhs;
 
 		/// number of feature vectors on left hand side
-		int32_t num_lhs;
+		index_t num_lhs;
 		/// number of feature vectors on right hand side
-		int32_t num_rhs;
+		index_t num_rhs;
 
```

Comment:
@vigsterkr Hi, Does all these changes for Kernel.h make sense to u? I just want to make sure these are necessary changes so I can change other places related to kernel. Thank you :)